src/util/text.ts

```diff
@@ -0,0 +1,19 @@
+import textCodeList from '~/codeLists/textCodeList';
```

Comment:
The naming needs to be changed. 
A file named text.ts, with a function named text doesn't tell that much

Reply:
CodeListHelper & getText